hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockOutputStream.java

```diff
@@ -84,7 +84,7 @@
  * through to the container.
  */
 public class BlockOutputStream extends OutputStream {
-  public static final Logger LOG =
```

Comment:
Won't this be private as well, or is the default access fine?

Reply:
This is used by subclasses.